app/views/exports/concentrations_ngul.csv.erb

```diff
@@ -1,10 +1,14 @@
 <%= CSV.generate_line ['Plate Barcode', @plate.labware_barcode.human], row_sep: "" %>
 <%# Essentially a unique element counter, ensuring that submissions will be numbered 1 upwards in the order we first saw them %>
 <% @pooling_ids = Hash.new {|h,i| h[i] = h.length + 1 } %>
-
+<% seen_well_locations = Set.new %><%# To prevent duplicated wells in the CSV at branch points %>
 <%= CSV.generate_line ['Well','Concentration','Pick','Pool'], row_sep: "" %>
 <% @plate.wells_in_columns.each do |well| %>
   <% well.submission_ids.each do |submission_id| %>
-<%= CSV.generate_line [well.position['name'], well.latest_concentration&.value, (well.passed? ? 1 : 0), @pooling_ids[submission_id]], row_sep: '' %>
+    <% well_location = well.position['name'] %>
+    <% pool_number = @pooling_ids[submission_id] %>
+    <% next if seen_well_locations.include?(well_location) %>
```

Comment:
So for our example yesterday with rows for pool 1 and 2, we now will only see 1 row with number 1? Is there a chance they will all have pool number 2 (if well.submissions_ids is not in the same order as @pooling_ids)? Which could be confusing.